src/components/Common/Breadcrumbs.tsx

```
@@ -0,0 +1,164 @@
+  }
+
+  if (breadcrumbs.length === 0) return null;
+
+  return (
+    <div className="flex items-center gap-4 mb-3">
+      <div className="flex items-center gap-2">
+        <ChevronLeft className="size-4" />
+        <span
+          onClick={() => goBack()}
+          className="text-sm text-gray-500 hover:cursor-pointer"
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)